py/kubeflow/testing/run_e2e_workflow.py

```diff
@@ -175,44 +204,19 @@ def run(args, file_handler): # pylint: disable=too-many-statements,too-many-bran
   workflow_names = []
   ui_urls = {}
 
-  for w in workflows:
+  for w in workflows: # pylint: disable=too-many-nested-blocks
     # Create the name for the workflow
     # We truncate sha numbers to prevent the workflow name from being too large.
     # Workflow name should not be more than 63 characters because its used
     # as a label on the pods.
     workflow_name = os.getenv("JOB_NAME") + "-" + w.name
-    ks_cmd = ks_util.get_ksonnet_cmd(w.app_dir)
-
-    # Print ksonnet version
-    util.run([ks_cmd, "version"])
 
     # Skip this workflow if it is scoped to a different job type.
     if w.job_types and not job_type in w.job_types:
       logging.info("Skipping workflow %s because job type %s is not one of "
                    "%s.", w.name, job_type, w.job_types)
       continue
 
-    # If we are scoping this workflow to specific directories, check if any files
```

Comment:
I think you want to leave lines 195-215 here. So that we always perform this check to decide whether or not to skip the test regardless of whether its a ksonnet or pyfunc workflow.

Reply:
Fixed.